### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 4

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#180] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152252279-1176476e-3916-4934-84be-5b08ad00cc32.png)

**Comment by Exyss** [comment]
1. $\binom{90}{6}$
2. $\binom{50}{3} \binom{40}{3}$
3. $\binom{50}{5} \binom{40}{1} = \binom{50}{5} \cdot 40$
4. $Tot - NessunoMate - NessunoInfo = \binom{90}{6} - \binom{50}{6} - \binom{40}{6}$
